### PR TITLE
adding my info to metadata.json

### PR DESCRIPTION
**Pull request by gottacatchenall**
added my info (name, affiliation, email, orcid) to metadata.json file

### CODE REVIEW

**Review by tpoisot — APPROVED** [review]

**Review by TanyaS08 — CHANGES_REQUESTED** [review]
Reorder - needs to be listed in same order as issues. 🐾

**Review by gottacatchenall — COMMENTED** [review]
flipped the order around

**Review by TanyaS08 — APPROVED** [review]
🌟

File: metadata.json [hunk 1]
```diff
@@ -12,6 +12,16 @@
...
       "orcid": "0000-0001-6067-1349"
     },
+    {
+      "affiliations": [
+        "McGill University",
+        "Québec Centre for Biodiversity Sciences"
+      ],
+      "familyname": "Catchen",
+      "givennames": "Michael David",
+      "email": "michael.catchen@mail.mcgill.ca",
```
Comment (tpoisot) [inline metadata.json:22]: remove the email for now

**Review by tpoisot — CHANGES_REQUESTED** [review]

**Review by gottacatchenall — COMMENTED** [review]
email removed

**Review by TanyaS08 — APPROVED** [review]
⭐

**Review by tpoisot — APPROVED** [review]

**Pull request merged**